apps/fio/readme.md

```diff
@@ -0,0 +1,38 @@
+# FIO
+
+Fio is a general I/O benchmark code, it has many options to configure and customize the I/O pattern being tested.
+
+To build a cluster with PBS, see one of the examples, e.g. [simple_hpc_pbs](../../examples/simple_hpc_pbs/readme.md)
+
+First log-in to the headnode.
+```
```

Comment:
You should replace the connect and manual run by something like azhpc-run -u build_fio.sh.
Look at the readme.md in ProLB.

Reply:
ok, will make this change.